notebooks/DAAC_UWG_cloud_tutorial/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -0,0 +1,430 @@
+    "5. The *nsidc-tutorials* environment is setup and activated. This [README](https://github.com/nsidc/NSIDC-Data-Tutorials/blob/main/README.md) has setup instructions.\n",
+    "\n",
+    "### **Example of end product** \n",
+    "At the end of this tutorial, the following figure will be generated:\n",
+    "<center>\n",
+    "<img src='./img/atl06_land_ice_heights_plot.png'/>\n",
+    "</center>\n",
+    "\n",
+    "### **Time requirement**\n",
+    "\n",
+    "Allow approximately 20 minutes to complete this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "24df578d",
+   "metadata": {},
+   "source": [
+    "## **2. Tutorial steps**\n",
+    "\n",
+    "## Import Packages\n",
+    "\n",
+    "The first step in any `python` script or notebook is to import packages.  In this tutorial requires the following pacakges:\n",
+    "- `earthaccess`, which enables Earthdata Login authentication and retrieves AWS credentials; enables collection and granule searches; and S3 access;\n",
+    "- `xarray`, used to load data;\n",
+    "- `hvplot`, used to visualize land ice height data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e1de126",
+   "metadata": {},
+   "source": [
+    "We are going to import the whole `earthaccess` package.\n",
+    "\n",
+    "We will also import the whole `xarray` package but use a standard short name `xr`, using the `import <package> as <short_name>` syntax.  We could use anything for a short name but `xr` is an accepted standard that most `xarray` users are familiar with.\n",
+    "\n",
+    "We only need the `xarray` module from `hvplot` so we import that using the `import <package>.<module>` syntax."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "127269e2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# For nice printing\n",
+    "from pprint import pprint\n",
+    "\n",
+    "# For searching NASA data\n",
+    "import earthaccess\n",
+    "\n",
+    "# For reading data, analysis and plotting\n",
+    "import xarray as xr\n",
+    "import hvplot.xarray"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed62342f",
+   "metadata": {},
+   "source": [
+    "## Authenticate\n",
+    "\n",
+    "The first step is to get the correct authentication that will allow us to get _cloud-hosted_ ICESat-2 data.  This is all done through Earthdata Login.  The `login` method also gets the correct AWS credentials.\n",
+    "\n",
+    "Login requires your Earthdata Login username and password.  This can be entered using an Environment Variable - the default strategy - or using either a`.netrc` file or interactively.  A `.netrc` file is a text file located in our home directory that contains login information for remote machines.  If we don't have a `.netrc` file `login` can create one for us.\n",
+    "\n",
+    "```\n",
+    "earthaccess.login(strategy='interactive', persist=True)\n",
```

Comment:
Should we be using .netrc here instead since that is what we suggest above, and to be consistent with our other notebooks? I know we also need this for our regression tests to run, but this one is going to be an exception given that it must be running in us-west-2 (a future problem that we need to solve).